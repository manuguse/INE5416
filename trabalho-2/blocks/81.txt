1 3 4 9 9 12 12 14
1 4 4 4 12 12 12 14
1 1 7 7 7 7 12 14
1 5 7 7 7 13 14 14
1 2 8 8 13 13 15 15
1 2 2 8 13 13 15 15
2 2 2 10 10 11 15 15
2 6 6 11 11 11 11 11