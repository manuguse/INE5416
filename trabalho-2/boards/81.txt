- - - - 1 5 3 5
- - - 4 6 - - 3
5 - - 3 - 7 - -
3 - 4 - - - - -
2 7 - - 2 - - 6
- - - - - - - -
5 - 3 - - - - 4
4 - - 3 5 - 6 1