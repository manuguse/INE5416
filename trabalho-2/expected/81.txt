7 1 3 2 1 5 3 5
6 2 1 4 6 4 2 3
5 4 6 3 5 7 1 2
3 1 4 2 1 5 4 1
2 7 1 3 2 4 3 6
1 2 6 2 1 3 2 5
5 1 3 1 2 4 1 4
4 2 1 3 5 2 6 1